### ISSUE TITLE
Web accessibility pass

### ISSUE THREAD

**Issue by caseyg** [caseyg/melchua#4]
Here's a really good checklist from Vox:

## Designers
- [x] Make sure there is enough contrast between text and its background color [More Info](http://accessibility.voxmedia.com/#designers-1)
- [x] Don't indicate important information using color alone [More Info](http://accessibility.voxmedia.com/#designers-2)
- [ ] Pair values of colors together (not only hues) to increase contrast [More Info](http://accessibility.voxmedia.com/#designers-3)
- [ ] Don't rely on sensory characteristics as the sole indicator for understanding and operating content [More Info](http://accessibility.voxmedia.com/#designers-4)
- [ ] Design focus states to help users navigate and understand where they are [More Info](http://accessibility.voxmedia.com/#designers-5)
- [ ] Help users understand inputs, and help them avoid and correct mistakes [More Info](http://accessibility.voxmedia.com/#designers-6)
- [x] Write good alt text for your images [More Info](http://accessibility.voxmedia.com/#designers-7)
- [x] If an experience cannot be made accessible, create another route for users to get that information [More Info](http://accessibility.voxmedia.com/#designers-8)
- [x] Be as consistent and clear as possible in layout and copy [More Info](http://accessibility.voxmedia.com/#designers-9)

## Engineers
- [x] Use the correct HTML element for your content [More Info](http://accessibility.voxmedia.com/#engineers-1)
- [x] Support keyboard navigation [More Info](http://accessibility.voxmedia.com/#engineers-2)
- [x] Understand and use HTML landmarks [More Info](http://accessibility.voxmedia.com/#engineers-3)
- [x] Write good alt text for your images [More Info](http://accessibility.voxmedia.com/#engineers-4)
- [x] Design focus states to help users navigate and understand where they are [More Info](http://accessibility.voxmedia.com/#engineers-5)
- [x] Help users understand inputs, and help them avoid and correct mistakes [More Info](http://accessibility.voxmedia.com/#engineers-6)
- [x] Use ARIA attributes when applicable [More Info](http://accessibility.voxmedia.com/#engineers-7)
- [x] Give users a way to skip top level navigation to access main content [More Info](http://accessibility.voxmedia.com/#engineers-8)
- [x] Make links descriptive [More Info](http://accessibility.voxmedia.com/#engineers-9)
- [x] Avoid images and iconography in pseudo-elements [More Info](http://accessibility.voxmedia.com/#engineers-10)
- [x] Make SVGs accessible to assistive technology [More Info](http://accessibility.voxmedia.com/#engineers-11)
- [ ] Hide decorative elements from screen readers [More Info](http://accessibility.voxmedia.com/#engineers-12)
- [ ] Create alternate routes for users to access information [More Info](http://accessibility.voxmedia.com/#engineers-13)
- [ ] Links should be visually identifiable and have clear :focus and :active states [More Info](http://accessibility.voxmedia.com/#engineers-14)
- [ ] HTML document should have a language attribute [More Info](http://accessibility.voxmedia.com/#engineers-15)

## Project Managers
- [ ] Familiarize yourself with the work associated with making content accessible [More Info](http://accessibility.voxmedia.com/#project-managers-1)
- [ ] Build in time for accessibility during project planning and sprint planning [More Info](http://accessibility.voxmedia.com/#project-managers-2)
- [ ] When sharing good work done by your team, praise efforts to increase accessibility [More Info](http://accessibility.voxmedia.com/#project-managers-3)
- [ ] The tools and products that you create should make accessibility easier to achieve [More Info](http://accessibility.voxmedia.com/#project-managers-4)
- [ ] Be an advocate for accessibility [More Info](http://accessibility.voxmedia.com/#project-managers-5)

## Quality Assurance
- [ ] Run through each page with the WAVE Chrome Extension. [More Info](http://accessibility.voxmedia.com/#qa-1)
- [ ] Users should be able to navigate through content using their keyboard. [More Info](http://accessibility.voxmedia.com/#qa-2)
- [ ] Users should be able to navigate content using a screen reader. [More Info](http://accessibility.voxmedia.com/#qa-3)
- [ ] The general architecture and hierarchy of the content should make sense. [More Info](http://accessibility.voxmedia.com/#qa-4)
- [ ] Charts and images should all have alt-text so that users with screen readers or users on a slow connection will still be able to understand the images. [More Info](http://accessibility.voxmedia.com/#qa-5)
- [ ] Decorative images should not be visible to screen readers. [More Info](http://accessibility.voxmedia.com/#qa-6)

## Editorial
- [ ] Write good alt text for your images [More Info](http://accessibility.voxmedia.com/#editorial-1)
- [ ] Editorial Engineers: think about how custom experiences can be made accessible [More Info](http://accessibility.voxmedia.com/#editorial-2)
- [ ] Be as consistent and clear as possible [More Info](http://accessibility.voxmedia.com/#editorial-3)
- [ ] Links should be descriptive [More Info](http://accessibility.voxmedia.com/#editorial-4)
- [ ] Important information should not be conveyed through images, color, or sensory characteristics alone [More Info](http://accessibility.voxmedia.com/#editorial-5)

## Tools
- [ ] Use these tools to help make your work accessible [More Info](http://accessibility.voxmedia.com/#tools-1)
 - [WCAG 2.0 guidelines to review best practices](http://code.viget.com/interactive-wcag/#responsibility=design&level=aa),
 - [WAVE (Web Accessibility Evaluation Tool) Chrome Extension to check accessibility across your products](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US),
 - [Web Accessibility Checker to check accessibility across your products](http://achecker.ca/checker/index.php),
 - [Color Contrast Analyzer App to analyze color contrast](https://www.paciellogroup.com/resources/contrastanalyser/),
 - [Color Palette Accessibility Checker to automatically check color combinations for passing contrast](https://accessibility.oit.ncsu.edu/tools/color-contrast/),
 - [Color Extractor bookmarklet to find all colors in CSS and load them in the Color Palette Accessibility Checker](https://accessibility.oit.ncsu.edu/tools/color-extractor/),
 - [Web Accessibility Checker to check accessibility across your products](http://achecker.ca/checker/index.php),
 - [Color Safe to make accessible color palettes](http://colorsafe.co/),
 - [Color Oracle to simulate color blindness](http://colororacle.org/index.html),
 - [Sim Daltonism to simulate color blindness](https://michelf.ca/projects/sim-daltonism/),
 - [Text over images accessibility tool](http://www.brandwood.com/a11y/)

**Comment by caseyg** [comment]
http://accessibility.voxmedia.com/

https://a11yproject.com/checklist
